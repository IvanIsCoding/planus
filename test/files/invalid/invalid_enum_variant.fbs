enum Foo: uint8 {
  X = "hello"
}