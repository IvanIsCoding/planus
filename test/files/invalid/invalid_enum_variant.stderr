[0m[1m[38;5;9merror[0m[1m: expecting integer literal[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_enum_variant.fbs:2:7
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   X = [0m[31m"hello"[0m
  [0m[34mâ”‚[0m       [0m[31m^^^^^^^[0m

Error: Bailing because of errors
